### PR TITLE
{lib}[GCCcore/8.2.0] RapidJSON v1.1.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/43f0f10f813ac8dc88f5ed83fe562924 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2419.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2be3b155a19d4f7e21d82abb14c6cd65 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3124.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c67d7907df93ebcedd78b44a7bb03bb6 for a full test report.

File: easybuild/easyconfigs/r/RapidJSON/RapidJSON-1.1.0-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'RapidJSON'
+version = '1.1.0'
+
+homepage = 'http://rapidjson.org'
```
Comment (verdurin) [inline easybuild/easyconfigs/r/RapidJSON/RapidJSON-1.1.0-GCCcore-8.2.0.eb]: @boegel Please fix the non-https URL

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/0ba909e90b98618b2151e20b4c012b1b for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/804688a8173d6f1dec9bc5378a780352 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**